src/grammar-registry.coffee

```diff
@@ -64,7 +63,8 @@ class GrammarRegistry extends FirstMate.GrammarRegistry
       fileTypes = fileTypes.concat(customFileTypes)
 
     for fileType, i in fileTypes
-      fileTypeComponents = fileType.toLowerCase().split(PathSplitRegex)
+      fileTypeComponents = fileType.toLowerCase().split("/")
```

Comment:
This isn't `path.sep` because it's supposed to match the file types listed in the grammar files, all of which use `/` for consistency.